src/tests/issues.targets

```diff
@@ -3664,4 +3664,10 @@
                 Condition="'%(ExcludeList.Extension)' == '.OutOfProcessTest'" />
       </ItemGroup>
     </Target>
+
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'mono' and '$(TargetArchitecture)' == 'riscv64'">
```

Comment:
This probably needs to be above the ``<Target Name="GetFilteredExcludeList" Returns="@(FilteredExcludeList)">`` block.